src/components/modals/modalStyles.scss

```diff
@@ -0,0 +1,34 @@
+$backGroundLight: white;
+$backgroundDark: rgba(0, 0, 0, 0.25);
+$primaryBorderColor: black;
```

Comment:
do you have global variables for colors somewhere? Should those go there as well?